neuralprophet/forecaster.py

```diff
@@ -811,6 +813,18 @@ def predict(self, df, decompose=True, raw=False):
             forecast = pd.concat((forecast, fcst), ignore_index=True)
         df = df_utils.return_df_in_original_format(forecast, received_ID_col, received_single_time_series)
         self.predict_steps = self.n_forecasts
+        # Conformal prediction interval with q
+        if self.q_hats:
+            if self.conformal_method == "naive":
+                df["yhat1 - qhat1"] = df["yhat1"] - self.q_hats[0]
+                df["yhat1 + qhat1"] = df["yhat1"] + self.q_hats[0]
+            else:  # self.conformal_method == "cqr"
+                quantile_hi = str(max(self.config_train.quantiles) * 100)
+                quantile_lo = str(min(self.config_train.quantiles) * 100)
+                df[f"yhat1 {quantile_hi}% - qhat1"] = df[f"yhat1 {quantile_hi}%"] - self.q_hats[0]
```

Comment:
in the case of CQR, I think my comment above also applies - let's simply overwrite `f"yhat1 {quantile_lo}%"` and  `f"yhat1 {quantile_hi}%"`.
IMO, When a user explicitly executes thes conformalize steps, it should be evident enough that the outcome is the conformalized quantile.

Reply:
I don't agree on this front. The only way it is evident is that is stating `f"yhat1 {quantile_hi}% - qhat1"`.